scrapper.py

```diff
@@ -44,20 +74,104 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(self.article_url, self.article_id)
+
+    def _fill_article_with_meta_information(self, article_bs):
+        self.article.author = 'NOT FOUND'
+
+        try:
+            post_tags = article_bs.find('div', class_='post-tags')
+            topics_bs = post_tags.find_all('a', class_='post-tags__tag')
+            self.article.topics = [topic_bs.text for topic_bs in topics_bs]
+        except AttributeError:
+            self.article.topics = 'NOT FOUND'
+
+        raw_date = article_bs.find('div', class_='post-date').text
+        months_dict = {'января': '01',
+                       'февраля': '02',
+                       'марта': '03',
+                       'апреля': '04',
+                       'мая': '05',
+                       'июня': '06',
+                       'июля': '07',
+                       'августа': '08',
+                       'сентября': '09',
+                       'октября': '10',
+                       'ноября': '11',
+                       'декабря': '12',
+                       }
+        for month in months_dict:
+            if month in raw_date:
+                raw_date = raw_date.replace(month, months_dict[month])
+        self.article.date = datetime.datetime.strptime(raw_date, '%d %m %Y, %H:%M')
+
+        self.article.title = article_bs.find('h1').text
+
+    def _fill_article_with_text(self, article_bs):
+        text = article_bs.find('div', class_='post-text').text
+        self.article.text = text
+
+    def parse(self):
+        response = requests.get(url=self.article_url,headers=HEADERS)
+
+        article_bs = BeautifulSoup(response.text, 'lxml')
+
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        return self.article
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    path_for_environment = Path(base_path)
+    if path_for_environment.exists():
+        shutil.rmtree(base_path)
+    path_for_environment.mkdir(parents=True)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path) as file:
+        configuration = json.load(file)
+
+    http_pattern = 'https://m.polit.ru'
+    for url in configuration["seed_urls"]:
+        if http_pattern not in url:
+            raise IncorrectURLError
+
+    seed_urls = configuration["seed_urls"]
+    total_articles_to_find_and_parse = configuration["total_articles_to_find_and_parse"]
+
+    if not seed_urls:
+        raise IncorrectURLError
+
+    if not isinstance(total_articles_to_find_and_parse, int) or total_articles_to_find_and_parse <= 0:
```

Comment:
if total_articles_to_find_and_parse is not an integer, this line will trigger TypeError before you will be able to raise IncorrectNumberOfArticlesError